src/main/resources/custom-buttons/custom_button_avm.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "id": "avm",
+        "name": "AVM for cBioPortal",
+        "tooltip": "Launch AVM for cBioPortal with data (copied to clipboard)",
+        "image_src": "https://aquminmedical.com/images/content/AquminLogoSimple.png",
+        "required_user_agent": "Win",
+        "required_installed_font_family": "AVMInstalled",
+        "url_format": "avm://?importclipboard&-AutoMode=true&-ProjectNameHint={studyName}&-ImportDataLength={dataLength}",
+        "visualize_title": "AVM for cBioPortal (Windows)",
+        "visualize_href": "https://bit.ly/avm-cbioportal",
+        "visualize_description": "Windows software that loads data into 3D Landscapes for interactive visualization and pathway analysis. Download table data directly from cBioPortal.",
+        "visualize_image_src": "https://github.com/user-attachments/assets/5c17f5ed-0357-4ffa-a6e1-5a9d435dd3c5"
+    }
+]
```

Comment:
Included in the "resources" folder so it can be easily enabled with one line (per the EXAMPLE)